spec/bdd/rspec_spec.rb

```diff
@@ -17,7 +17,11 @@
 
         context.example("a complex example") do
           Given("a condition X") { }
-          And("a condition Y") {}
+          And("a condition Y") {
+            Then("overwrite happened") {
+              But("it did nothing")
```

Comment:
this was showing `SKIPPED` without indentation after `a condition Y` - it's fixed now :)